packages/cli/src/lib/commands.ts

```diff
@@ -9,8 +9,8 @@ export const commands: CommandModule[] = [
     ...yargsAutorunCommandObject(),
     command: '*',
   },
-  yargsAutorunCommandObject(),
+  yargsConfigCommandObject(),
   yargsCollectCommandObject(),
   yargsUploadCommandObject(),
-  yargsConfigCommandObject(),
+  yargsAutorunCommandObject(),
```

Comment:
If this is about the help output, then I stick to the original ordering. The `autorun` command is the main command for most users (possibly the only command they will use), whereas `print-config` is more of a helper command for debugging (probably more useful to us than to our users).

Reply:
Ok makes more sense to take the user perspective!